Azure/ARO-RP
# Use image digests for fluentbit images

### Which issue this PR addresses:

Fixes [ARO-4826](https://issues.redhat.com/browse/ARO-4826)

### What this PR does / why we need it:

If we accidentally push over the existing fluentbit image tag in production, we risk breaking all cluster creations.  

Using a digest for the fluentbit image will prevent that from happening because the digest should change if the underlying container image changes.  So we can push the same tag to our heart's content.  

### Test plan for issue:

- Check release pipelines to ensure the fluentbit digests are the same
- Update RP config
- Deploy to INT
- Profit

### Is there any documentation that needs to be updated for this PR?

Yes, fluentbit publishing will be updated.
